packages/nimble-components/src/table-column/date-text/types.ts

```diff
@@ -0,0 +1,107 @@
+/**
```

Comment:
I had seen that definition, but I'm not sure where that gets me. Are you imagining something like `public customDay: typeof DateTimeFormatOptions.day;`? For some reason, I can't get that interface to resolve, but assuming I could, I'm not sure that is what we'd want.

Reply:
Yeah, that's more or less what I was envisioning. If TypeScript had exposed its underlying types then I was hoping we could either use them directly or use them to construct our types. That way our types would update automatically if new options were added. But I also don't see a way to do that if we can't access those interfaces or their member types.

It could be reasonable to file an issue to TypeScript asking for those types to be exported. I think it's a valid use case to want to provide an API that wraps the native API and not have to re-implement the types. But not critical to do immediately.

Reply:
Did a little StackOverflow searching to find the right syntax, but I got it working. Now we define types like this:
`export type WeekdayFormat = Intl.DateTimeFormatOptions['weekday'];`

Reply:
Awesome! You just made my day 😁